### ISSUE TITLE
Find a new way to generate leads

### ISSUE THREAD

**Issue by blalor** [tofuutils/tenv#245]
**Describe the bug**
You spammed 450+ people.

**To Reproduce**
Please don’t. 

**Expected behavior**
Never ever getting unsolicited email about this project. 

**Screenshots**
![image](https://github.com/user-attachments/assets/8ef10b25-f96d-4302-a288-fb5f0bc4f6c3)

**Environment (please complete the following information):**
- OS: all of them
- tofuenv version: 3.1.0 (apparently)

**Additional context**
https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies#4-spam-and-inauthentic-activity-on-github

**Comment by wzyboy** [comment]
It is considered impolite to send unsolicited bulk emails by harvesting email addresses from GitHub.

It it considered **rude** to put all your recipients in the "To" field instead of Bcc.

I would do my best to tell all my teammates NOT to use anything from tofuutils.

**Comment by kvendingoldo** [comment]
Yes, it was a mistake in our email delivery system. We use only BCC all times for all updates. Sorry for our mistake.

**Comment by blalor** [comment]
> Yes, it was a mistake in our email delivery system. We use only BCC all times for all updates. Sorry for our mistake.

You missed the point. The core problem isn’t that you didn’t use BCC, it’s that you harvested hundreds of emails and spammed them.

**Comment by kvendingoldo** [comment]
I didn’t miss it. The delivery is disabled, no more worries about it.
